mars/oscar/backends/ray/communication.py

```diff
@@ -273,9 +280,15 @@ async def handle_task(message: Any, object_ref: "ray.ObjectRef"):
                 try:
                     result = await object_ref
                 except Exception as e:  # pragma: no cover
+                    # The error ClientObjectRef can't be formatted, so
+                    # we give it a string `ClientObjectRef` instead.
+                    try:
+                        object_ref_str = str(object_ref)
+                    except Exception:
+                        object_ref_str = "ClientObjectRef"
```

Comment:
```suggestion
                        object_ref_str = repr(object_ref)
```

Reply:
Any operations on the object_ref will raise an exception, this is the fallback path, we'd better to use a plain text here.